tests/Makefile

```diff
@@ -114,6 +114,7 @@ DIRS += dotnet-sos
 DIRS += tkillself
 DIRS += thread_abort
 DIRS += synccall
+DIRS += python_flask_tests
```

Comment:
Running an existing test suite of a python module qualifies as a "solution". Suggest to move to solutions directory.

Reply:
Will move to solution